File: ios/WebViewFramework/WebViewFramework/WebContainerConfig.swift
//
//  WebContainerConfig.swift
//  WebViewFramework
//
//  Created by Willy on 2025/5/26.
//
struct Profile: Codable {
    let userId: String
    let email: String
    let deviceId: String
}

struct WebContainerConfig: Codable {
    let authToken: String
    let region: String
    let language: String
    let userAgent: String
    let profile: Profile
}

File: android/WebViewFramework/app/src/main/java/com/gogolook/webviewframework/WebContainerConfig.kt
package com.gogolook.webviewframework

data class WebContainerConfig(
    val authToken: String,
    val region: String,
    val language: String,
    val userAgent: String,
    val profile: Profile
)

data class Profile(
    val userId: String,
    val email: String,
    val deviceId: String
)
